ui/src/app/components/help-sidebar.tsx

```diff
@@ -177,7 +194,7 @@ const icons = (helpContentKey: string) => {
     faIcon: helpContentKey === NOTEBOOK_HELP_CONTENT ? faFolderOpen : faInfoCircle,
     label: helpContentKey === NOTEBOOK_HELP_CONTENT ? 'Storage Icon' : 'Help Icon',
     page: null,
-    style: {fontSize: '21px'},
+    style: {fontSize: '1.1rem'},
```

Comment:
This style is only applied to the info icon

Reply:
If you use inspect mode on the Invision screens, looks like both should be ~21px font size